packages/python/src/components/Variable.tsx

```diff
@@ -0,0 +1,24 @@
+import { Children, code } from "@alloy-js/core";
+import { usePythonNamePolicy } from "../name-policy.js"; // assuming you have this
+
+export interface VariableProps {
+  name: string;
+  value?: Children;
+  type?: Children; // Optional, only for type annotation
+  omitNone?: boolean; // Optional, to omit None assignment
```

Comment:
This configuration seems a bit strange. Is it the case that if I provide a type, that none may not be the best default? I feel like it's ok to let initialization be up to the caller.